lib/StatusIndicator/index.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const StatusIndicator = props => 
+  <span>
+    <i className="fa fa-circle" style={{ color: props.color }}></i> { props.label }
```

Comment:
I really like the use of `props.label` here.

If there's going to be some design difference between the `label` and `children` prop then I recommend we wrap the label in a span and maybe give it a element className.